web/src/components/DeferredOnboardingQuestion.tsx

```diff
@@ -96,7 +58,21 @@ export const DeferredOnboardingQuestion = (props: Props): ReactElement => {
           onBack: (): void => {},
         }}
       >
-        {props.isTaskPage ? onTaskPage : onDashboard}
+        <div className="padding-3">
+          {props.label}
+          <div className="display-flex mobile-lg:flex-row flex-column mobile-lg:flex-align-center">
+            <div className="margin-right-1 width-100 text-field-width-default margin-bottom-05">
+              {props.children}
+            </div>
+            <SecondaryButton
+              isColor="primary"
+              onClick={onSubmit}
+              dataTestId="deferred-question-save"
```

Comment:
Lower then dust: With that cleanup I guess we dont need the isRightMarginRemoved anymore?